Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -126,46 +124,44 @@ Developers should take care to secure sensitive IPC components with the `signatu
 #### References
 
 ##### OWASP Mobile Top 10 2016
-* M1 - Improper Platform Usage - https://www.owasp.org/index.php/Mobile_Top_10_2016-M1-Improper_Platform_Usage
+
+-	M1 - Improper Platform Usage - https://www.owasp.org/index.php/Mobile_Top_10_2016-M1-Improper_Platform_Usage
 
 ##### OWASP MASVS
-* V6.1: "The app only requires the minimum set of permissions necessary."
+
+-	V6.1: "The app only requires the minimum set of permissions necessary."
 
 ##### CWE
-* CWE-250 - Execution with Unnecessary Privileges
+
+-	CWE-250 - Execution with Unnecessary Privileges
 
 ##### Info
-* [1] Android Permissions - https://developer.android.com/guide/topics/permissions/requesting.html
-* [2] Custom Permissions - https://developer.android.com/guide/topics/permissions/defining.html
-* [3] An In-Depth Introduction to the Android Permission Model - https://www.owasp.org/images/c/ca/ASDC12-An_InDepth_Introduction_to_the_Android_Permissions_Modeland_How_to_Secure_MultiComponent_Applications.pdf
-* [4] Android Permissions - https://developer.android.com/reference/android/Manifest.permission.html#ACCESS_LOCATION_EXTRA_COMMANDS
+
+-	[1] Android Permissions - https://developer.android.com/guide/topics/permissions/requesting.html
+-	[2] Custom Permissions - https://developer.android.com/guide/topics/permissions/defining.html
+-	[3] An In-Depth Introduction to the Android Permission Model - https://www.owasp.org/images/c/ca/ASDC12-An_InDepth_Introduction_to_the_Android_Permissions_Modeland_How_to_Secure_MultiComponent_Applications.pdf
+-	[4] Android Permissions - https://developer.android.com/reference/android/Manifest.permission.html#ACCESS_LOCATION_EXTRA_COMMANDS
 
 ##### Tools
-* AAPT - http://elinux.org/Android_aapt
-* Drozer - https://github.com/mwrlabs/drozer
 
+-	AAPT - http://elinux.org/Android_aapt
+-	Drozer - https://github.com/mwrlabs/drozer
 
 ### Testing Input Validation and Sanitization
 
 #### Overview
 
-Android apps can expose functionality to:
-* other apps via IPC mechanisms like Intents, Binders, Android Shared Memory (ASHMEM) or BroadcastReceivers,
-* through custom URL schemes (which are part of Intents) and
-* the user via the user interface.
+Android apps can expose functionality to:* other apps via IPC mechanisms like Intents, Binders, Android Shared Memory (ASHMEM) or BroadcastReceivers,* through custom URL schemes (which are part of Intents) and* the user via the user interface.
```

Comment:
Yes, the editor did something weird. Can you please help by doublechecking? I thought i fixed it all now..